.eslintrc.js

```diff
@@ -52,6 +52,24 @@ module.exports = {
         'react/prop-types': [0],
         'react/jsx-boolean-value': [2, 'always'],
         'import/prefer-default-export': 'off',
+        // TODO: remove
+        '@typescript-eslint/camelcase': 'off',
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
+        '@typescript-eslint/no-unsafe-member-access': 'off',
+        '@typescript-eslint/no-unsafe-call': 'off',
+        '@typescript-eslint/no-unsafe-assignment': 'off',
+        '@typescript-eslint/no-unsafe-return': 'off',
+        '@typescript-eslint/ban-ts-comment': 'off',
+        '@typescript-eslint/restrict-template-expressions': 'off',
+        'no-console': 'off',
+        '@typescript-eslint/ban-types': 'off',
+        '@typescript-eslint/no-floating-promises': 'off',
+        '@typescript-eslint/no-implied-eval': 'off',
+        '@typescript-eslint/no-use-before-define': 'off',
+        '@typescript-eslint/prefer-as-const': 'off',
+        '@typescript-eslint/restrict-plus-operands': 'off',
+        'no-shadow': 'off',
+
```

Comment:
to be reverted individually in future PRs

Reply:
each rule has been added to a checklist to be done individually https://trello.com/c/5z3uBARR/2442-typescript-eslint-rule-fixes